ntc_templates/templates/huawei_vrp_display_version.textfsm

```diff
@@ -6,4 +6,5 @@ Value UPTIME (.+)
 
 Start
   ^.*software,\s+Version\s+${VRP_VERSION}\s+\(${PRODUCT_VERSION}\)
-  ^HUAWEI\s+${MODEL}\s+uptime\s+is\s+${UPTIME}$$
\ No newline at end of file
+  ^HUAWEI\s+${MODEL}\s+uptime\s+is\s+${UPTIME}$$
+  ^Huawei\s+${MODEL}\s+Router\s+uptime\s+is\s+${UPTIME}$$

```

Comment:
A single line with:
`  ^Huawei\s+${MODEL}\s+(Router\s+)?uptime\s+is\s+${UPTIME}$$`
maybe better?
I am surprised how it passes the other tests